app/components/market/local-template-browser/local-template-browser.scss

```diff
@@ -9,7 +9,7 @@ bl-local-template-browser {
 
     > .header {
         height: 40px;
-        padding: 0 20px;
+        padding: 10px 20px;
     }
```

Comment:
padding for the local template heading between the breadcrumb and then heading to bring it in line with the other NCJ screens.